swagger2.0/oic.wk.scenecollection.swagger.json

```diff
@@ -0,0 +1,273 @@
+              "links": [
+                {"href": "/scenemember1","rt": ["oic.wk.scenemember"],"if":["oic.if.baseline"]},
+                {"href": "/scenemember2","rt": ["oic.wk.scenemember"],"if":["oic.if.baseline"]}
+              ]
+            },
+            "schema": {
+              "$ref": "#/definitions/SceneCollection"
+            }
+          }
+        }
+      },
+      "post": {
+        "description": "Provides the action to change the last set scene selection.\nCalling this method shall update  all scene members to the prescribed membervalue.\nWhen this method is called with the same value as the current lastScene value\nthen all scene members shall be updated.\n",
+        "parameters": [
+          {
+            "$ref": "#/parameters/interface-update"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SceneCollectionUpdate"
+            },
+            "x-example": {
+              "lastScene": "Reading"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description" : "Indicates that the value is changed.\nThe changed properties are provided in the response.\n",
+            "x-example": {
+              "lastScene": "Reading"
+            },
+            "schema": {
+              "$ref": "#/definitions/SceneCollectionUpdate"
+            }
+          }
+        }
+      }
+    },
+    "/SceneCollectionResURI?if=oic.if.ll" : {
+      "get": {
+        "description": "Collection that models a set of Scenes.\nThis resource is a generic collection resource with additional parameters.\nThe rts value shall contain oic.scenemember resource types.\nThe additional parameters are\n  lastScene, this is the scene value last set by any OCF Client\n  sceneValues, this is the list of available scenes\n  lastScene shall be listed in sceneValues.\n",
+        "parameters": [
+          {
+            "$ref": "#/parameters/interface-ll"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description" : "",
+            "x-example": [
+              {"href": "/scenemember1","rt": ["oic.wk.scenemember"],"if":["oic.if.baseline"]},
+              {"href": "/scenemember2","rt": ["oic.wk.scenemember"],"if":["oic.if.baseline"]}
+            ],
+            "schema": {
+              "$ref": "#/definitions/slinks"
+            }
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "interface-update" : {
+      "in" : "query",
+      "name" : "if",
+      "type" : "string",
+      "enum" : ["oic.if.a","oic.if.baseline"]
+    },
+    "interface-baseline" : {
+      "in" : "query",
+      "name" : "if",
+      "type" : "string",
+      "enum" : ["oic.if.baseline"]
+    },
+    "interface-ll" : {
+      "in" : "query",
+      "name" : "if",
+      "type" : "string",
+      "enum" : ["oic.if.ll"]
+    },
+    "interface-all" : {
```

Comment:
Where is this referenced as the place to find the Default OCF Interface?  The way I read the schema, the Default OCF Interface would be included in "interface-baseline".

Reply:
DMWG still hasn't provided finalized guidance for how we're doing this.  So until that point The defaults for Core Resources are as per Spec (for the most part).